src/olympia/reviewers/models.py

```diff
@@ -430,7 +430,14 @@ def get_event(cls, addon, status, version=None, post_review=False,
                     'No such version/auto approval summary when determining '
                     'event type to award points: %r', exception)
                 weight = 0
-            if weight > amo.POST_REVIEW_WEIGHT_HIGHEST_RISK:
+
+            if addon.type == amo.ADDON_DICT:
+                reviewed_score_name = 'REVIEWED_DICT_FULL'
```

Comment:
Technically... couldn't it be an update ? If auto-approval is disabled on some extension, because of the hack below we're passing `post_review=True`, sure, but that could still be an update right ?

Reply:
Yes, I am simplifying things here since we are giving the same points for updates and initial submissions and the whole thing will probably be rewritten sooner or later anyway. Also, in post-review we generally don't distinguish (as much) between initial submissions and updates anymore.